README.md

```diff
@@ -51,6 +51,28 @@ this tool. This will isolate the dependencies and ensure they don't conflict wit
 $ pipx install git-co-evg-base
 ```
 
+### Debugging installation issues
```

Comment:
Would we want to add this to the documentation instead (or in addition)? it might be worth having an FAQ page. I worry about it get lost in the README.